content/dataset/expectation-issue.md

```diff
@@ -2,13 +2,13 @@
 attribution: crown-copyright
 collection: ''
 dataset: expectation-issue
-description: issues found checking expecatations
+description: Issues found checking expecatations
 end-date: ''
 entity-maximum: ''
 entity-minimum: ''
 entry-date: ''
 fields:
-- field: response-id
+- field: result-id
```

Comment:
can we rename to result not result-id. This follows the rest of the schema i.e. endpoint is the hash identifier for an endpoint. entity is the entity number for an entity.